src/qt/transactionrecord.cpp

```diff
@@ -94,10 +94,23 @@ QList<TransactionRecord> TransactionRecord::decomposeTransaction(const CWallet *
 
         if (fAllFromMe && fAllToMe)
         {
+            std::string address;
+            if (wtx.tx->vout.size() == 2)
+            {
+                for (const CTxOut& txout : wtx.tx->vout)
+                {
+                    if (wallet->IsChange(txout)) continue;
```

Comment:
Not sure if its worth do add a call to IsChange for every transaction decomposing with the new interface structure...